networkx/algorithms/shortest_paths/unweighted.py

```diff
@@ -176,6 +208,18 @@ def all_pairs_shortest_path_length(G, cutoff=None):
     >>> length[2][2]
     0
 
+    Only include paths with length less than or equal to the `cutoff` keyword
+    argument:
+
+    >>> length = dict(nx.all_pairs_shortest_path_length(G, cutoff=2))
+    >>> for node in [0, 1, 2, 3, 4]:
+    ...     nlen = length[1][node] if node in length[1] else "N/A"
+    ...     print(f"1 - {node}: {nlen}")
+    1 - 0: 1
+    1 - 1: 0
+    1 - 2: 1
+    1 - 3: 2
+    1 - 4: N/A
```

Comment:
Can this shorter example be adapted to the other examples too?
```suggestion
    >>> path_lengths = dict(nx.all_pairs_shortest_path_length(G, cutoff=2))
    >>> path_lengths[1]  # node 4 is too far away to appear
    {0: 1, 1: 0, 2: 1, 3: 2}
```